backend/src/controllers/bars.ts

```diff
@@ -58,7 +58,7 @@ const createBarController = async (
     arrayOfBeverages.forEach((e) => {
       if (!verifyBeverageCategory(e.category)) {
         shouldContinue = false;
-        return res.status(400).send("this beverage category doesn't exist");
+        return res.status(403).send("this beverage category doesn't exist");
```

Comment:
403 c'est forbidden donc ca s'applique pas ici!

Reply:
Bah si justement, pour moi c'est le plus pertinent... La requête contient une value qui n'est pas accepté, donc on l'interdit

Reply:
Y a des bonnes doc dessus. https://httpstatuses.com/403 ce site en général permet de voir quels sont les codes à appliquer en fct des situations:
en général je fais:
- 400 il manque des params ou ils sont pas bons
- 403 auth credentials pas bonne ou manquante
- et là où il y a débat c'est dans le cadre d'une API, s'il on ne trouve pas d'item associé à l'API alors on fait soit un 422 (ce que j'avais commencé à faire) soit un 404 (not found, le truc classique des pages web)